app/views/glossary_terms/_form.html.erb

```diff
@@ -10,50 +10,10 @@
     <%= f.text_area :description, rows: 20, class: "form-control" %>
   </div>
 
-  <%# Currently, allow image only when creating term %>
-  <% if action == :new %>
-    <div class="form-group mt-3">
-      <%= f.label :upload_image, :Image.t + ":" %>
-      <%= custom_file_field :upload_image, { id: "customFileField" } %>
-      <%# f.file_field :upload_image %>
-      <%# tag.div class: "input-group mb-3" do
-            # tag.div class: "input-group-prepend" do
-            #   tag.span "Upload", class: "input-group-text", id: "customFileFieldLabel"
-            # end
-            tag.div class: "custom-file" do
-              custom_file_field :upload_image, { id: "customFileField" }
-              # tag.label :no_file_selected.t, class: "custom-file-label", for: "customFileField"
-            end
-          end %>
-
-    </div>
-
-    <div class="form-group mt-3 form-inline">
-      <%= f.label :copyright_holder,
-                  :glossary_term_copyright_holder.t + ":" %><br/>
-      <%= f.text_field :copyright_holder,
-                       value: @copyright_holder,
-                       class: "form-control" %>
-      <%# f.select_year @copyright_year,
-                        { field_name: :copyright_year,
-                          start_year: 1980,
-                          end_year: Time.now.year },
-                        { class: "form-control" } %>
-      <%= f.date_select @copyright_year,
-                        { field_name: :copyright_year,
-                          start_year: 20.years.ago.year,
-                          end_year: Time.now.year,
-                          discard_day: true,
-                          discard_month: true },
-                        { class: "form-control" } %><br>
-      <%# f.select :upload, :license_id, @licenses, selected: @upload_license_id,
-                   class: "form-control" %>
-      <%= f.select :license_id, @licenses,
-                   { selected: @upload_license_id },
-                   { class: "custom-select" } %>
-      <p class="help_block">(<%= :glossary_term_copyright_warning.t %>)</p>
-    </div>
-  <% end %>
+  <%# Let view add image input form if needed %>
+  <%# Currently, an image is added only when creating, not editing, term.
+      See /config/locales/en.txt :glossary_term_index_intro %>
+  <%= yield f %>
```

Comment:
This lets us eliminate the conditional and the `action` flag.

Reply:
Thanks for figuring this out. I will study what you did and try to apply as I go through!

Right now, i'm doing a whirlwind tour to add the @navbar to all views, so we can start to pass all those tests that relied on the tabset links.